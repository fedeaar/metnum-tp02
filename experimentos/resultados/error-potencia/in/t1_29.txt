-6.962958376223297137e+02 4.446106886788871293e+01 1.263188672502534757e+02 2.266577494291292965e+02 2.463813140371156294e+02 6.097848987221350825e+01 1.734302115585574313e+02 4.396277598009246645e+01 2.206981881931089262e+01 1.072979111501559970e+02 2.920055279720708086e+02 1.677656143272834077e+00 2.406714707155295940e+01 6.475847447219068442e+01 9.078792162399464871e+01 1.961939922143970705e+02 1.084103272513103491e+02 4.293560696991154657e+01 8.240174075247630014e+01 3.683399959817325708e+01
4.446106886788871293e+01 8.084920730204073607e+02 -5.247115199605040203e+01 -7.932575524333569206e+01 -7.628552772845299756e+00 -1.569320317375794787e+02 -6.973348851216294975e+01 -2.942041149270997735e+01 -3.290767835683011810e+01 -4.687896969864057439e+01 -3.744642393556114257e+01 -1.518994806519633700e+00 -6.919202883220402889e+00 -1.929160783380770567e+02 -6.805636386090232293e+01 -9.761090391016729484e+01 -3.164810859546678845e+00 -7.754919034611850748e+01 -1.020838164845106775e+02 -3.730549012626672045e+00
1.263188672502534473e+02 -5.247115199605040203e+01 1.396088108561068566e+02 2.808869532456852625e+01 7.853121165465128684e+01 -7.524365801910320783e+01 1.597427915934241938e+01 -3.122137497675363083e+00 -1.264375960205053140e+01 7.601405416132990922e+00 7.572733777295036361e+01 -3.611419867120703175e-01 3.900868032932128582e+00 -9.594153803393326996e+01 -1.090540993880330944e+01 6.636635732851058123e+00 3.467169015866357995e+01 -3.285949029025938017e+01 -3.451632631610571877e+01 1.015874388301361542e+01
2.266577494291292680e+02 -7.932575524333570627e+01 2.808869532456852625e+01 1.217698395639592945e+02 1.465917989418212244e+02 -1.140006392797847923e+02 4.080074938145997976e+01 -1.040393261821980886e+00 -1.810343803482609104e+01 2.162118471152319188e+01 1.462045378477336897e+02 -4.238171121153435639e-01 8.335073875271385546e+00 -1.465169265175176463e+02 -9.224303778686776312e+00 2.800678189947725016e+01 6.468784708068862699e+01 -4.833279530813249636e+01 -4.744783700861142961e+01 1.940494750438083216e+01
2.463813140371156294e+02 -7.628552772845300645e+00 7.853121165465128684e+01 1.465917989418212244e+02 -2.705323044054957222e+02 -1.252040066179695366e+01 1.087039511119767923e+02 2.305514383237868614e+01 4.623339413727343583e+00 6.582130896441945822e+01 2.136678941495207482e+02 7.279458579439470611e-01 1.614178987705252055e+01 -2.335785029816558378e+01 4.481401461294584010e+01 1.157967827618926293e+02 8.344125619811643446e+01 3.810077502949499983e+00 2.522617982023034600e+01 2.733289439499432660e+01
6.097848987221347983e+01 -1.569320317375795071e+02 -7.524365801910317941e+01 -1.140006392797847354e+02 -1.252040066179695188e+01 7.081561939960540712e+02 -1.000363686563441092e+02 -4.200261252903430176e+01 -4.679333838805291634e+01 -6.718592451000910160e+01 -5.509785617532255486e+01 -2.164515880609405585e+00 -9.973520063583187323e+00 -2.738704842969522701e+02 -9.708633006004680510e+01 -1.397050882626226667e+02 -5.237221859829113590e+00 -1.102087814342837504e+02 -1.452555530635632408e+02 -5.542740835638644725e+00
1.734302115585574313e+02 -6.973348851216296396e+01 1.597427915934242293e+01 4.080074938145997976e+01 1.087039511119767923e+02 -1.000363686563441092e+02 1.388208689043379422e+02 -3.576643648789946894e+00 -1.664600282321945102e+01 1.167853640419394168e+01 1.055770400654146499e+02 -4.609427389058332603e-01 5.563573459090365603e+00 -1.277342681873009980e+02 -1.336294411329017606e+01 1.161709632722811136e+01 4.798792245961992364e+01 -4.346066889442808190e+01 -4.513505878891500345e+01 1.413064364956455421e+01
4.396277598009246645e+01 -2.942041149270997735e+01 -3.122137497675363971e+00 -1.040393261821982218e+00 2.305514383237868614e+01 -4.200261252903430176e+01 -3.576643648789946894e+00 3.424797001929907765e+02 -7.850587429279982210e+00 -3.362471996172569089e+00 1.858383988975166901e+01 -2.944402341735476991e-01 3.522711380440819995e-01 -5.268555872971456466e+01 -1.158120162152023624e+01 -9.807816711263686926e+00 1.020353033352873950e+01 -1.943593826099692023e+01 -2.291644525987319270e+01 2.649748146553488226e+00
2.206981881931089262e+01 -3.290767835683011810e+01 -1.264375960205053140e+01 -1.810343803482609104e+01 4.623339413727344471e+00 -4.679333838805292345e+01 -1.664600282321945102e+01 -7.850587429279985763e+00 7.094507889466096913e+02 -1.145374772793379847e+01 -3.303055338688980491e+00 -4.221118966288778251e-01 -1.457309795931413410e+00 -5.781202270433828261e+01 -1.846938672837605822e+01 -2.462025600172515638e+01 2.093628929306989406e+00 -2.276049178565675390e+01 -2.922788403585401440e+01 -1.096303228652921735e-01
1.072979111501559970e+02 -4.687896969864057439e+01 7.601405416132988258e+00 2.162118471152319188e+01 6.582130896441945822e+01 -6.718592451000913002e+01 1.167853640419394168e+01 -3.362471996172567312e+00 -1.145374772793379670e+01 1.562136890989270057e+02 6.271639616181712285e+01 -3.416778539713281604e-01 3.105464314240881318e+00 -8.548707074119421634e+01 -1.087423244443541925e+01 3.130069427358265433e+00 2.906533986038836304e+01 -2.956678124001283692e+01 -3.157497015339129121e+01 8.445771501573785756e+00
2.920055279720708086e+02 -3.744642393556113547e+01 7.572733777295036361e+01 1.462045378477336897e+02 2.136678941495207482e+02 -5.509785617532254065e+01 1.055770400654146641e+02 1.858383988975166901e+01 -3.303055338688981379e+00 6.271639616181712285e+01 -3.054840922397380254e+01 4.331817134466130481e-01 1.657173676198635448e+01 -7.679922498858110202e+01 3.329358587588754403e+01 1.063940802718583853e+02 9.414968433093534372e+01 -1.584822965901661185e+01 2.141737811751697418e+00 3.013949090166785538e+01
1.677656143272834077e+00 -1.518994806519633700e+00 -3.611419867120703175e-01 -4.238171121153436749e-01 7.279458579439472832e-01 -2.164515880609406473e+00 -4.609427389058332603e-01 -2.944402341735478656e-01 -4.221118966288778251e-01 -3.416778539713283269e-01 4.331817134466134922e-01 4.909827709899640809e+02 -2.226021250619709263e-02 -2.695752842532800120e+00 -7.184468228178836346e-01 -8.046083236536399896e-01 3.232047110041838534e-01 -1.025791879306790166e+00 -1.261737005025441505e+00 6.965893931433544961e-02
2.406714707155296296e+01 -6.919202883220402889e+00 3.900868032932129026e+00 8.335073875271387323e+00 1.614178987705251700e+01 -9.973520063583187323e+00 5.563573459090365603e+00 3.522711380440822215e-01 -1.457309795931412522e+00 3.105464314240882207e+00 1.657173676198635803e+01 -2.226021250619706487e-02 -1.897947294121702555e+01 -1.295727989241706979e+01 6.979023660215791236e-02 4.606861944438136902e+00 7.119837424382926550e+00 -4.054010559640828681e+00 -3.568701978929716301e+00 2.179846056598198523e+00
6.475847447219068442e+01 -1.929160783380770852e+02 -9.594153803393329838e+01 -1.465169265175176463e+02 -2.335785029816558378e+01 -2.738704842969522701e+02 -1.277342681873009980e+02 -5.268555872971456466e+01 -5.781202270433828261e+01 -8.548707074119420213e+01 -7.679922498858110202e+01 -2.695752842532800120e+00 -1.295727989241706624e+01 6.427528350711961593e+02 -1.214240021211658984e+02 -1.768770050874092874e+02 -9.945996687233291667e+00 -1.358635535742830314e+02 -1.799154499070109523e+02 -7.970555309254194754e+00
9.078792162399464871e+01 -6.805636386090232293e+01 -1.090540993880331300e+01 -9.224303778686770983e+00 4.481401461294584010e+01 -9.708633006004681931e+01 -1.336294411329017962e+01 -1.158120162152023624e+01 -1.846938672837605822e+01 -1.087423244443541925e+01 3.329358587588754403e+01 -7.184468228178836346e-01 6.979023660215613600e-02 -1.214240021211658984e+02 3.719901784876764737e+02 -2.818138895257536447e+01 1.985247502761109573e+01 -4.537074044434678655e+01 -5.445805178570463312e+01 4.892537177639673551e+00
1.961939922143970421e+02 -9.761090391016732326e+01 6.636635732851053682e+00 2.800678189947725016e+01 1.157967827618926293e+02 -1.397050882626226667e+02 1.161709632722811669e+01 -9.807816711263688703e+00 -2.462025600172514928e+01 3.130069427358266321e+00 1.063940802718584138e+02 -8.046083236536401007e-01 4.606861944438136014e+00 -1.768770050874092874e+02 -2.818138895257536447e+01 2.018087590727100462e+02 5.116009465265664602e+01 -6.258880622703842533e+01 -6.935564792595123151e+01 1.449901213738820971e+01
1.084103272513103491e+02 -3.164810859546679733e+00 3.467169015866357995e+01 6.468784708068864120e+01 8.344125619811643446e+01 -5.237221859829117143e+00 4.798792245961992364e+01 1.020353033352874128e+01 2.093628929306990294e+00 2.906533986038836304e+01 9.414968433093534372e+01 3.232047110041837978e-01 7.119837424382926550e+00 -9.945996687233296996e+00 1.985247502761109928e+01 5.116009465265664602e+01 -4.272529527550935882e+02 1.813998265024436396e+00 1.128722521366657894e+01 1.204198406447249248e+01
4.293560696991154657e+01 -7.754919034611849327e+01 -3.285949029025938017e+01 -4.833279530813251057e+01 3.810077502949493766e+00 -1.102087814342837220e+02 -4.346066889442808900e+01 -1.943593826099692023e+01 -2.276049178565675390e+01 -2.956678124001283692e+01 -1.584822965901661185e+01 -1.025791879306790166e+00 -4.054010559640828681e+00 -1.358635535742830314e+02 -4.537074044434678655e+01 -6.258880622703843954e+01 1.813998265024436396e+00 7.450216310462546971e+02 -7.008101394771128412e+01 -1.287234382464958138e+00
8.240174075247630014e+01 -1.020838164845107059e+02 -3.451632631610571877e+01 -4.744783700861142961e+01 2.522617982023031757e+01 -1.452555530635632408e+02 -4.513505878891498924e+01 -2.291644525987319270e+01 -2.922788403585402150e+01 -3.157497015339128765e+01 2.141737811751683207e+00 -1.261737005025441727e+00 -3.568701978929716301e+00 -1.799154499070109523e+02 -5.445805178570463312e+01 -6.935564792595123151e+01 1.128722521366658604e+01 -7.008101394771128412e+01 5.481801737156470153e+02 1.240454280167008427e+00
3.683399959817325708e+01 -3.730549012626675598e+00 1.015874388301361364e+01 1.940494750438083216e+01 2.733289439499432305e+01 -5.542740835638643837e+00 1.413064364956455421e+01 2.649748146553488226e+00 -1.096303228652926176e-01 8.445771501573787532e+00 3.013949090166785538e+01 6.965893931433544961e-02 2.179846056598198523e+00 -7.970555309254191201e+00 4.892537177639672663e+00 1.449901213738820971e+01 1.204198406447249425e+01 -1.287234382464958138e+00 1.240454280167007539e+00 -3.101193360147317435e+02
