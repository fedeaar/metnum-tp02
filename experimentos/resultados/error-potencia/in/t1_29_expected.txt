-9.900000000000000000e+02
9.790000000000000000e+02
9.310000000000000000e+02
9.190000000000000000e+02
7.990000000000000000e+02
7.190000000000000000e+02
6.370000000000000000e+02
4.910000000000000000e+02
-4.640000000000000000e+02
-4.600000000000000000e+02
4.010000000000000000e+02
3.470000000000000000e+02
-3.140000000000000000e+02
-2.640000000000000000e+02
2.090000000000000000e+02
1.510000000000000000e+02
1.290000000000000000e+02
1.150000000000000000e+02
5.700000000000000000e+01
-2.000000000000000000e+01
